### DISCUSSION TITLE
Graph: Showing value of point

### DISCUSSION THREAD

**Discussion by baur** [grafana/grafana#43157] [Dashboards, Panels & Visualizations]
Hi, it would be very nice to show the value of point 

<img width="577" alt="image" src="https://user-images.githubusercontent.com/2898361/146200609-907381f3-5691-4fb0-91f5-12bdb6702817.png">

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
